kotlin-asyncapi-annotation/src/main/kotlin/org/openfolder/kotlinasyncapi/annotation/channel/MessageExample.kt

```diff
@@ -1,10 +1,8 @@
 package org.openfolder.kotlinasyncapi.annotation.channel
 
-import org.openfolder.kotlinasyncapi.annotation.Schema
-
 annotation class MessageExample(
     val default: Boolean = false,
-    val headers: Schema = Schema(default = true),
+    val headers: String = "",
```

Comment:
Why was this changed to String?

Reply:
I initially made a mistake. this field holds an object and not a schema. see the integration test for clarification.